src/Database/Driver.php

```diff
@@ -289,11 +289,11 @@ public function supportsCTEs(): bool
     /**
      * @inheritDoc
      */
-    public function quote($value, $type = PDO::PARAM_STR): string
+    public function quote(string $value, int $type = PDO::PARAM_STR): string
     {
         $this->connect();
 
-        return $this->_connection->quote((string)$value, $type);
+        return $this->_connection->quote($value, $type);
```

Comment:
This no longer casts to string. I'm not sure we should be quoting non-string values here.